### PR TITLE
(maint) Update for facter 3

### PR DESCRIPTION
**Pull request by kylog**
* Remove cfacter references as it's not shipped anymore
* Update to indicate facter is now an exe

### CODE REVIEW

File: file_paths.md [hunk 1]
```diff
@@ -99,7 +97,6 @@ The files annotated by an '*' indicate that they are created by package installa
             libssl.so *
             ruby *
                 vendor_ruby *             # ruby code
-                    cfacter.rb *
                     facter.rb *
```
Comment (joshcooper) [inline file_paths.md]: just to double check, the `facter.rb` shim exists?

Comment (ferventcoder) [inline file_paths.md]: I would believe something exists here. A quick install could prove this?

Comment (whopper) [inline file_paths.md]: Just verified it does in the latest puppet-agent build.

Comment (kylog) [inline file_paths.md]: Also, FYI it is a generated file, which is why you won't find it in the facter repo. However, you would find https://github.com/puppetlabs/facter/blob/8886a003ac6459e56d78cd3d2d98c6d1e8b1898b/lib/facter.rb.in

**Pull request merged**